src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,61 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../react-app-env';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+type OnSelectTab = (id: string, tab: Tab) => void | undefined;
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.some(tab => tab.id === selectedTabId)
+    ? selectedTabId
+    : tabs[0].id;
+  const onSelectTab: OnSelectTab = (id, tab) => {
+    if (selectedTabId !== id) {
+      onTabSelected(tab);
+    }
+  };
+
+  const selectedTabContent = tabs
+    .find(tab => selectedTabId === tab.id)?.content;
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const { id, title } = tab;
```

Comment:
To me it seems that you don't really need destructuring here, those names are pretty short and clear. If you don't desctructure it  you'll have `tab.id`and `tab.title`, what doesn't look bad at all